modules/navier_stokes/src/fvbcs/WCNSFVMomentumFluxBC.C

```diff
@@ -71,7 +71,12 @@ WCNSFVMomentumFluxBC::computeQpResidual()
     if (MooseUtils::absoluteFuzzyEqual(*_area_pp, 0))
       mooseError("Surface area is 0");
 
-  const Real contrib = std::abs(_face_info->normal()(_index));
+  // We assume that this boundary condition can only be specified on a boundary side.
+  // This means that the normal on the side points outwards and since the mass flow is
+  // defined as an absolute quantity, we need to use the negative of the normal here.
```

Comment:
What guarantee do we have that it is an absolute quantity?

Reply:
I'll rewrite this comment. I think the way I put it is wrong. We want to have a positive value for inflow and a negative value for outflow. so we need to use the negative of the normal if we are on a boundary. If we are not on a boundary then I have no idea on how to ensure that the inflow is really an inflow. Maybe we can ask for a direction parameter from the user. So I think for now we should restrict this.

Reply:
Okay, I'll remove the boundary restriction and add another parameter which can be used to define the inlet direction. If it is specified, we use that, if not, we use the normal (whatever it is).